tests/AbstractEditableRepositoryTest.php

```diff
@@ -84,6 +85,26 @@ public function testAddFile()
         $this->assertSame(TestFile::BODY, $file->getBody());
     }
 
+    public function testAddLink()
```

Comment:
I think we should test what happens when accessing the children of a link when a link points to a directory.